tech/languages/nodejs/nodejs.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Node.js
+page: nodejs
+section: tech-languages
+order: 1
+version: 0.10.36
+---
+
+# Node.js
+
+## npm
+
+To install npm, simply type:
+
+```
+$ sudo dnf install npm
+```
+
+This will install v8 engine, nodejs runtime and npm package manager and its dependencies.
```

Comment:
And I would either merge npm+modules into one page or reference (link) the module installation from here.